packages/cadl-ranch-specs/http/payload/content-negotiation/main.tsp

```diff
@@ -54,7 +54,7 @@ namespace DifferentBody {
 
   model PngImageAsJson {
     @header contentType: "image/jpeg";
-    content: bytes;
+    content: string;
   }
```

Comment:
This is an expected fix

Reply:
@timotheeguerin 

Should it be
```ts
  model PngImageAsJson {
    @header contentType: "application/json";
    content: bytes;
  }
```
(or without `contentType`)

Reply:
e.g. https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/ContentSafety/models.tsp#L102-L104